build/generate_bundle.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 set -e
-REL=$(dirname "$0")
+set -x
+
+LOGFILE=${LOGFILE:-/dev/null}
+# If LOGFILE is /dev/null, this command fails, so ignore that error
+truncate --size=0 ${LOGFILE} || true
+
+OPERATOR_SDK=${OPERATOR_SDK:-operator-sdk}
```

Comment:
This will also be used downstream in CPaaS, so we'll need to keep an eye on this to make sure it's also compatible there.